Makefile

```diff
@@ -109,6 +109,24 @@ ISTIO_DOCKER:=${ISTIO_OUT}/docker_temp
 # dir where tar.gz files from docker.save are stored
 ISTIO_DOCKER_TAR:=${ISTIO_OUT}/docker
 
+# Populate the git version for istio/proxy (i.e. Envoy)
+ifeq ($(PROXY_TAG),)
+  export PROXY_TAG:=$(shell grep PROXY_TAG istio.VERSION  | cut -d '=' -f2 | tr -d '"')
```

Comment:
removing source of truth from machine parsable json to a shell script is going backwards.

Is it much easier to update `istio.deps` and everyone else reads dependencies from it.

Reply:
@mandarjog I don't disagree ... although my understanding of all of this machinery is not great.  Feel free to roll a follow-on PR and I'd be happy to review.